### PR TITLE
Modified messages

### PR DESCRIPTION
**Pull request by MadhuriUttarkar**

### CODE REVIEW

File: src/main/java/org/madhuri/app/message/dao/MessageDAO.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package org.madhuri.app.message.dao;
+import java.sql.Connection;
```
Comment (MadhuriUttarkar) [inline src/main/java/org/madhuri/app/message/dao/MessageDAO.java:3]: space between package and import statements

File: src/main/java/org/madhuri/app/message/handler/LoginHandler.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    @POST
+    public Response login(Profile profile) {
+        Optional<Profile> foundProfile = profileDAO.findByUsername(profile.getUsername());
+
+        if (foundProfile.isPresent()) {
+            if (foundProfile.get().getPassword().equals(profile.getPassword())) {
+            	System.out.println("Profile found");
+                return Response.ok(foundProfile).build();
```
Comment (MadhuriUttarkar) [inline src/main/java/org/madhuri/app/message/handler/LoginHandler.java]: foundProfile will have username & password, do you want to return them in response ? empty HTTP 200 response should work

File: src/main/java/org/madhuri/app/message/model/Message.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
+package org.madhuri.app.message.model;
+
+import java.util.Date;
+import jakarta.json.bind.annotation.JsonbProperty;
+import jakarta.json.bind.annotation.JsonbDateFormat;
+
+
+public class Message {
```
Comment (MadhuriUttarkar) [inline src/main/java/org/madhuri/app/message/model/Message.java:8]: no sender & recipient details ?
which user is this message going to ?

File: src/main/java/org/madhuri/app/message/handler/LoginHandler.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    public LoginHandler() {
+        this.profileDAO = new ProfileDAO();
+    }
+
+    @POST
+    public Response login(Profile profile) {
+        Optional<Profile> foundProfile = profileDAO.findByUsername(profile.getUsername());
+
+        if (foundProfile.isPresent()) {
+            if (foundProfile.get().getPassword().equals(profile.getPassword())) {
```
Comment (MadhuriUttarkar) [inline src/main/java/org/madhuri/app/message/handler/LoginHandler.java]: you are already checking password in login service, why are you using profile DAO in login handler directly ? 
you can use login service right ?

**Review by MadhuriUttarkar — COMMENTED** [review]
take a look at the comments.

File: src/main/java/org/madhuri/app/message/mockdata/MockDataClass.java [hunk 5]
```diff
@@ -2,14 +2,14 @@
 import java.util.HashMap;
 import java.util.Map;
```
Comment (dwarakauttarkar) [inline src/main/java/org/madhuri/app/message/mockdata/MockDataClass.java]: is this class still required ?

Comment (MadhuriUttarkar) [reply src/main/java/org/madhuri/app/message/mockdata/MockDataClass.java]: removed the class

File: src/main/java/org/madhuri/app/message/model/Message.java (hunk 3 above)
Comment (MadhuriUttarkar) [reply src/main/java/org/madhuri/app/message/model/Message.java:8]: Added recipient in the message class

**Review by dwarakauttarkar — APPROVED** [review]

**Pull request merged**